packages/components/src/Button/Button.js

```diff
@@ -35,11 +52,12 @@ function Button(props, forwardedRef) {
 			if (
 				isWithinButtonGroup &&
 				enableSelectNone &&
-				isButtonGroupActive
+				isButtonGroupActive &&
+				buttonGroup
 			) {
 				event.preventDefault();
 				event.stopPropagation();
-				buttonGroup.setState(null);
+				buttonGroup.setState(undefined);
```

Comment:
`setState` actually doesn't accept `null` but it does accept `undefined` 🙂

Reply:
TIL!